priv/schemas/bundle_config_schema_v3.yaml

```diff
@@ -0,0 +1,103 @@
+---
+"$schema": http://json-schema.org/draft-04/schema#
+title: Bundle Config v3
+description: A config schema for bundles
+type: object
+required:
+  - cog_bundle_version
+  - name
+  - version
+  - commands
+additionalProperties: false
+properties:
+  cog_bundle_version:
+    type: number
+    enum:
+      - 2
```

Comment:
The plan @mpeck and I discussed on Friday was to auto-convert v2 compliant configs if the commands are already multi calling convention since converting a non-enforcing command is simply generating a rule `when command is foo:bar allow`.